sunpy/coordinates/tests/test_utils.py

```diff
@@ -323,3 +324,25 @@ def test_rectangle_bottom_left_vector():
     assert bottom_left.spherical.lat == bottom_left_vector[0].spherical.lat
     assert top_right.spherical.lon == bottom_left_vector[1].spherical.lon
     assert top_right.spherical.lat == bottom_left_vector[1].spherical.lat
+
+
+def test_solar_angle_equivalencies_inputs():
+
+    with pytest.raises(TypeError):
+        solar_angle_equivalency("earth")
+
+    test_coord = SkyCoord(0*u.arcsec, 0*u.arcsec)
+    with pytest.raises(ValueError):
+        solar_angle_equivalency(test_coord)
+
+
+def test_solar_angle_equivalencies_outputs():
+
+    observer = get_body("earth", Time("2020-11-16"))
```

Comment:
```suggestion
    observer = get_earth("2020-11-16")
```
It's cleaner to call our `get_earth()`.